### PR TITLE
Display map and markers

### LINKED ISSUE #16: Create Map Display
Create a JS function to initialize the Google Map that will be the focal point of the Maptivist website.

### LINKED ISSUE #17: Display Markers
Create JS function to create new markers and display them on the Map

### PR DESCRIPTION
**Pull request by TBarrett553**
Right now the marker position keeps returning null.

### CODE REVIEW

File: maptivist/src/main/webapp/script.js [hunk 1]
```diff
@@ -12,27 +12,149 @@
...
+function createInfowindow(position) {
+  //Set info window content from form
+  var title = document.getElementById('marker-title').value;
+  var location = position.toString();
+  var description = document.getElementById('marker-description').value;
+  var link = document.getElementById('marker-link').value;
+  var category = document.getElementsByClassName('marker-category').value;
+  
+  // Set the content of the info window 
+  var contentString = `<div id="marker-window"><h1>${title}</h1><br><h2>${location}</h2><br><p>${description}</p><br><p>${link}</p><br>`
```
Comment (flying-bus) [inline maptivist/src/main/webapp/script.js]: I think template literals (the ``) allow newlines. Could you format the HTML to be more readable with newlines? (e.g. after all of the closing tags?

File: maptivist/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,27 +12,149 @@
...
+        }
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+        // Add the result contents to the infowindow display
+        infowindowContent.children['place-icon'].src = place.icon;
+        infowindowContent.children['place-name'].textContent = place.name;
+        infowindowContent.children['place-address'].textContent = address;
+        infowindow.open(map);
```
Comment (flying-bus) [inline maptivist/src/main/webapp/script.js]: The indentation looks wrong here compared to the previous line.

File: maptivist/src/main/webapp/script.js [hunk 3]
```diff
@@ -12,27 +12,149 @@
...
+    }
+    
+    // Concatentates a readable address for an autocomplete result
+    var address = '';
+        if (place.address_components) {
+            address = [
+                (place.address_components[0] && place.address_components[0].short_name || ''),
+                (place.address_components[1] && place.address_components[1].short_name || ''),
+                (place.address_components[2] && place.address_components[2].short_name || '')
+            ].join(' ');
```
Comment (flying-bus) [inline maptivist/src/main/webapp/script.js]: Would it just be simpler to use "formatted_address"?

https://developers.google.com/maps/documentation/geocoding/intro#Results

**Review by flying-bus — APPROVED** [review]
A couple suggestions, but overall looks good!

File: maptivist/src/main/webapp/script.js (hunk 2 above)
Comment (TBarrett553) [reply maptivist/src/main/webapp/script.js]: Ok yeah I see what you mean, they shouldn't be indented.

File: maptivist/src/main/webapp/script.js (hunk 1 above)
Comment (TBarrett553) [reply maptivist/src/main/webapp/script.js]: Ok sure I'll try that!

File: maptivist/src/main/webapp/script.js (hunk 3 above)
Comment (TBarrett553) [reply maptivist/src/main/webapp/script.js]: Ok that makes sense and should be a simple fix!

**Review by kanbi18 — CHANGES_REQUESTED** [review]
The form action is "/marker" to be POST.

**Review by kanbi18 — APPROVED** [review]

**Pull request merged**